test-result-summary-client/src/utils/perf.js

```diff
@@ -1,9 +1,6 @@
+import {fetchData} from "./Utils";
 
 export const getBenchmarkMetricProps = async (benchmarkName) => {
-    let parserProps = await fetch( `/api/getBenchmarkMetricProps?benchmarkName=${benchmarkName}`, {
-            method: 'get'
-        }
-    );
-    parserProps = await parserProps.json();
+    let parserProps = await fetchData( `/api/getBenchmarkMetricProps?benchmarkName=${benchmarkName}`);
     return parserProps;
 }
```

Comment:
Since we simplify `getBenchmarkMetricProps()` to one line, I am not sure we need this function anymore. I think we can explicitly write the fetchData() at
https://github.com/adoptium/aqa-test-tools/blob/060754f6cbf64857055e55446f791525c35ac323/test-result-summary-client/src/PerfCompare/PerfCompare.jsx#L447
https://github.com/adoptium/aqa-test-tools/blob/060754f6cbf64857055e55446f791525c35ac323/test-result-summary-client/src/TabularView/TabularView.jsx#L541

Reply:
So should I just remove the `utils/perf.js` file? (As it only has one function which we wish to remove anyway)

Reply:
yes, we should just remove it.

Reply:
Removed it!